lib/epoch.ex

```diff
@@ -1,21 +1,36 @@
 defmodule DateTimeParser.Epoch do
   @moduledoc false
 
-  import NimbleParsec
+  @token_key :unix_epoch
+
   import DateTimeParser.Formatters, only: [format: 1]
-  import DateTimeParser.Combinators.Epoch
 
-  defparsec(:parse, unix_epoch())
+  def parse(%{"seconds" => raw_seconds, "subseconds" => raw_subseconds}) do
+    with {seconds, _} <- Integer.parse(raw_seconds) do
```

Comment:
Let's also ensure nothing is lingering. Eg, if this function gets "100a" as a second, then `Integer.parse("100a") #=> {100, "a"}`. We should match on `{100, ""}` to ensure the regex isn't messing up somehow.